src/Model.php

```diff
@@ -1465,32 +1434,28 @@ public function tryLoadAny()
      *
      * @return $this
      */
-    public function loadBy(string $field_name, $value)
+    public function loadBy(string $fieldName, $value)
     {
-        // store
-        $field = $this->getField($field_name);
-        $system = $field->system;
-        $default = $field->default;
+        $field = $this->getField($fieldName);
 
-        // add condition and load record
-        $this->addCondition($field_name, $value);
+        // backup
```

Comment:
I would remove backup/restore comment, also in tryLoadBy

Reply:
Done